build.gradle

```diff
@@ -214,13 +214,17 @@ else {
 def arguments = []
 def jvmArguments = []
 
-if(usesMixins.toBoolean()) {
+if (usesMixins.toBoolean()) {
     arguments += [
-            "--tweakClass org.spongepowered.asm.launch.MixinTweaker"
-    ]
-    jvmArguments += [
-            "-Dmixin.debug.countInjections=true", "-Dmixin.debug.verbose=true", "-Dmixin.debug.export=true"
+        "--tweakClass org.spongepowered.asm.launch.MixinTweaker"
     ]
+    if (usesMixinDebug.toBoolean()) {
```

Comment:
Can you check earlier if `usesMixinDebug` is set and if not set it to the same as `useMixins` - this will preserve the current operation if the buildscript is applied to a project with an existing config. Something like:

```groovy
if (!project.hasProperty(usesMixinDebug)) {Boolean usesMixinDebug = useMixins}
```

Reply:
Sure